README.asciidoc

```diff
@@ -1,5 +1,9 @@
 == Glossary of Gradle terms
 
+[[artifact]]
+=== Artifact
+'Artifact' is usually used in the context of publishing to designate a file whose name has a 'classifier' and 'extension'. Two artifacts in a publication cannot have the same classifier and extension.
```

Comment:
I wonder if this should also include a definition for "publication", since that also has special meaning in the context of the `maven-publish` plugin?

Reply:
Agreed, let's close this pull request and I'll consolidate everything in https://github.com/autonomousapps/gradle-glossary/pull/6